packages/react/src/components/DesignLibrary/DesignLibraryApp.tsx

```diff
@@ -18,6 +18,7 @@ export const DesignLibraryApp = ({
   page,
   componentMap,
   loadServerImportMap,
+  loadClientImportMap,
```

Comment:
From my understanding we don't need to add `loadClientImportMap` prop and pass it to `DesignLibrary`, since client import map is provided here in [Providers.tsx](https://github.com/Sitecore/content-sdk/blob/dev/packages/create-content-sdk-app/src/templates/nextjs-app-router%20(beta)/src/Providers.tsx#L13). Also you need to update Providers value since `import-map` file is not present anymore in App Router solution

Reply:
HOC takes import map from context, but I also don't see any place where import map is posted to context. It should be fine to pass it this way, IMO?
@yavorsk what's the right approach based on your recent work?

Reply:
@art-alexeyenko As I mentioned in a previous comment it's passed [here](https://github.com/Sitecore/content-sdk/blob/dev/packages/create-content-sdk-app/src/templates/nextjs-app-router%20(beta)/src/Providers.tsx#L13), you need to update it to reference client import map, default import map is not present anymore. So `loadClientImportMap` prop is not needed

Reply:
I'd leave the loadClientImportMap prop just for simplicity. The current approach is kind of over-complicated with one server map passed directly, then client map applied in a separate place via context + HOC. At least the override should be easier to follow.

Reply:
@art-alexeyenko It’s not simplified - it has actually become more complex. Why should we support an additional property if the map is already available through the provider? This is exactly how it works in AppPlaceholder. Why should customers supply a client map here if it’s already provided by the Providers? Could you please simplify this? There is no reason to keep it and it's not logical